rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -31,9 +31,48 @@
         <server2.jvm.args>
             -Djboss.socket.binding.port-offset=${server.port.offset} ${jvm.args.other} ${jvm.args.memory} ${jvm.args.modular}
         </server2.jvm.args>
+        <version.quarkus>1.4.2.Final</version.quarkus>
+        <version.smallrye.jwt>2.1.1</version.smallrye.jwt>
+        <version.httpclient>4.5.12</version.httpclient>
+        <version.microprofile.jwt.auth.api>1.1.1</version.microprofile.jwt.auth.api>
+        <version.microprofile.config.api>1.4</version.microprofile.config.api>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>io.smallrye</groupId>
+            <artifactId>smallrye-jwt</artifactId>
+            <version>${version.smallrye.jwt}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>${version.httpclient}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.microprofile.jwt</groupId>
+            <artifactId>microprofile-jwt-auth-api</artifactId>
+            <version>${version.microprofile.jwt.auth.api}</version>
```

Comment:
should not be this in scope `provided`? The Quarkus could still load it but in case of WildFly there should not be packed into the compiled war of the coordinator.

Reply:
Yes sure, I will make the change